src/track.ts

```diff
@@ -44,5 +41,4 @@ function track(action, label, duration) {
 
 // New code should import track by name so that it can be stubbed
 // in unit tests. Legacy code still imports as default.
-track.track = track;
-module.exports = track;
+export default track;
```

Comment:
We have generally avoided default exports when converting to TS, instead preferring to change all importing files to use `import { track } from "../track"` style instead.

Reply:
Done.